tools/dynamodbConnector.py

```diff
@@ -0,0 +1,103 @@
+import boto3
+import logging
+from botocore.exceptions import ClientError
+
+
+class DynamoDBConnector:
+    """Connector to make the web scraper compatible with AWS Dynamodb. It relies
+    on boto3 and uses the credentials stored in ~/.aws/credentials.
+    """
+
+    def __init__(self):
+        """Initialise the connection, try to create the tables if they don't
+        exist yet. Create a logger instance as well.
+        """
+        self.logger = logging.getLogger(__name__)
+        self.dynamodb = boto3.resource(
+            'dynamodb',
+            region_name='eu-west-2',
```

Comment:
Similar to the comment above, this configuration should live in terraform right?